<launch>
  <arg name="no_multi_range" default="false" />
  <arg name="show_plots" default="false" />
  <arg name="ignore_z_position" default="true" />
  <arg name="tracker_frame" default="vicon/UWB_Solo/UWB_Solo" />
  <arg name="target_frame" default="uwb/target" />
  <include file="$(find uwb)/launch/uwb_multi_range.launch" unless="$(arg no_multi_range)" />
  <node name="uwb_tracker_node" pkg="uwb" type="uwb_tracker_node.py" output="screen">
    <param name="show_plots" value="$(arg show_plots)" />
    <param name="ignore_z_position" value="$(arg ignore_z_position)" />
    <param name="tracker_frame" value="$(arg tracker_frame)" />
    <param name="target_frame" value="$(arg target_frame)" />
  </node>
</launch>
